src/app-layer-ssl.h

```diff
@@ -176,9 +178,17 @@ typedef struct SSLState_ {
 
     /* there might be a better place to store this*/
     uint16_t hb_record_len;
+
+    uint8_t finished;
+
+    DetectEngineState *de_state;
+
+    uint16_t events;
```

Comment:
Looking at this, I think 'finished' could be a flag instead. Then events and hb_record_len could fill the gap right after the flags.